src/htmlminifier.js

```diff
@@ -395,6 +400,17 @@ function processScript(text, options, currentAttrs) {
   return text;
 }
 
+function minifyJson(text) {
+  try {
+    // Try parsing and re-stringifying the JSON
+    return JSON.stringify(JSON.parse(text));
+  }
+  catch (e) {
```

Comment:
Is there a way to log the error instead of suppressing it?  Similar to what is done here: https://github.com/kangax/html-minifier/blob/ab4c9d6e3e11ab207f545728c918895d5cb4c5f8/src/htmlminifier.js#L740C11-L740C28